service/src/test/kotlin/app/cash/backfila/actions/CreateBackfillActionTest.kt

```diff
@@ -35,18 +35,32 @@ class CreateBackfillActionTest {
   @MiskTestModule
   val module: Module = BackfilaTestingModule()
 
-  @Inject lateinit var configureServiceAction: ConfigureServiceAction
-  @Inject lateinit var createBackfillAction: CreateBackfillAction
-  @Inject lateinit var scope: ActionScope
-  @Inject lateinit var queryFactory: Query.Factory
-  @Inject @BackfilaDb lateinit var transacter: Transacter
-  @Inject lateinit var fakeBackfilaClientServiceClient: FakeBackfilaClientServiceClient
+  @Inject
```

Comment:
Is this needed?

Reply:
intellij keeps reformatting this way

Reply:
I'm getting so sick of these formatting changes.

Reply:
I should probably pull in the square style guide.. but also I think intellij doesn't let you configure this particular one